--- a/src/gui/main_window.rs
+++ b/src/gui/main_window.rs
@@ -50,8 +50,8 @@
 const WINDOW_BORDER_WIDTH: f64 = 1.0;
 const PADDING_X: f64 = 5.0;
 const PADDING_Y: f64 = 10.0;
-const ITEM_WIDTH: f64 = 210.0;
-const ITEM_HEIGHT: f64 = 32.0;
+const ITEM_WIDTH: f64 = 280.0;
+const ITEM_HEIGHT: f64 = 44.0;
 
 impl FocusData for UIState {
     fn has_autofocus(&self) -> bool {
@@ -241,7 +241,7 @@
     if cfg!(target_os = "macos") {
         32.0
     } else if cfg!(target_os = "linux") {
-        24.0
+        32.0
     } else {
         24.0
     }
@@ -416,7 +416,7 @@
         Vec2::ZERO,
     );
 
-    let text_size = 10.0;
+    let text_size = 13.0;
     let font = FontDescriptor::new(FontFamily::MONOSPACE)
         .with_weight(FontWeight::NORMAL)
         .with_size(text_size);
--- a/src/gui/ui_theme.rs
+++ b/src/gui/ui_theme.rs
@@ -543,8 +543,8 @@
             cursor_color: Color::WHITE,
             about_background_color: Color::rgb8(27, 32, 32),
             font_family: FontFamily::SYSTEM_UI,
-            browser_label_size: 12.0,
-            profile_label_size: 11.0,
+            browser_label_size: 15.0,
+            profile_label_size: 14.0,
             url_label_size: 12.0,
             label_color_hover: Color::rgb8(255, 255, 255),
             secondary_label_color_hover: Color::rgb8(255, 255, 255),
@@ -575,8 +575,8 @@
             cursor_color: Color::BLACK,
             about_background_color: Color::rgb8(236, 236, 236),
             font_family: FontFamily::SYSTEM_UI,
-            browser_label_size: 12.0,
-            profile_label_size: 11.0,
+            browser_label_size: 15.0,
+            profile_label_size: 14.0,
             url_label_size: 12.0,
             label_color_hover: Color::rgb8(0, 0, 0),
             secondary_label_color_hover: Color::rgb8(0, 0, 0),
